ovos_stt_plugin_fasterwhisper/__init__.py

```diff
@@ -244,20 +246,28 @@ def available_languages(self) -> set:
 }
 
 if __name__ == "__main__":
-    b = FasterWhisperSTT(config={"model": "projecte-aina/faster-whisper-large-v3-ca-3catparla"})
+    print(FasterWhisperSTT.MODELS)
```

Comment:
Do you want to keep the print?

Reply:
it's just debug code, wont show unless you run the file directly so leave it